src/test/rpc_tests.cpp

```diff
@@ -346,4 +360,61 @@ BOOST_AUTO_TEST_CASE(rpc_getnetworksolps)
     BOOST_CHECK_NO_THROW(CallRPC("getnetworksolps 120 -1"));
 }
 
+// Test parameter processing (not functionality)
+BOOST_AUTO_TEST_CASE(rpc_insightexplorer)
+{
+    CheckRPCThrows("getaddressmempool \"a\"",
+        "Error: getaddressmempool is disabled. "
+        "Run './zcash-cli help getaddressmempool' for instructions on how to enable this feature.");
+    CheckRPCThrows("getaddressutxos \"a\"",
+        "Error: getaddressutxos is disabled. "
+        "Run './zcash-cli help getaddressutxos' for instructions on how to enable this feature.");
+    CheckRPCThrows("getaddressdeltas \"a\"",
+        "Error: getaddressdeltas is disabled. "
+        "Run './zcash-cli help getaddressdeltas' for instructions on how to enable this feature.");
+    CheckRPCThrows("getaddressbalance \"a\"",
+        "Error: getaddressbalance is disabled. "
+        "Run './zcash-cli help getaddressbalance' for instructions on how to enable this feature.");
+    CheckRPCThrows("getaddresstxids \"a\"",
+        "Error: getaddresstxids is disabled. "
+        "Run './zcash-cli help getaddresstxids' for instructions on how to enable this feature.");
+
+    fExperimentalMode = true;
+    fInsightExplorer = true;
```

Comment:
Revert these at the end of the test (e.g. this test would fail if it were run immediately after itself, due to mutation of global state).